openapi/connector_mgmt.yaml

```diff
@@ -1080,6 +1421,134 @@ components:
               items:
                 $ref: "#/components/schemas/ConnectorType"
 
+    #
+    # Connector Namespaces
+    #
+    ConnectorNamespaceRequestMeta:
+      type: object
+      properties:
+        name:
+          type: string
+        annotations:
```

Comment:
on the [kas](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/c2e0d76158fe955b227e200491bbaf06b33cd8a9/openapi/kas-fleet-manager-private.yaml#L240) side, annotations are defined as a key/value as you would expect on kubernetes. They have a fixed set of properties and we don't but I think we should use the same syntax as kubernetes.

Reply:
It's optional.

Reply:
it is optional but it leaks into the api so people may want to sue it
if it make sense, we should extend this to any request I guess.

Reply:
I changed it to comply with k8s style as you suggested. Can you elaborate what you mean by "extend this to any request?"

Reply:
we could extend the support for annotations to any resource which could be handy for clients to attach metadata to the resource they create. i.e. you can attach some attributes to a `ConnectorRequest`

Reply:
Yeah, it's good architectural design and complies with k8s resource model. I'll create a JIRA ticket for it. We can do it after preview release if we don't have time to get to it before that.

Reply:
Created issue https://issues.redhat.com/browse/MGDCTRS-632